doc/nvim-tree-lua.txt

```diff
@@ -888,6 +894,10 @@ Configuration for diagnostic logging.
         File copy and paste actions.
           Type: `boolean`, Default: `false`
 
+        *nvim-tree.log.types.dev*
```

Comment:
I keep needing a "clean" log context that doesn't get removed by `validate_options`. Feel free to remove.